pymedphys/_dicom/anonymise.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (C) 2018 Matthew Jennings, Simon Biggs
+# Copyright (C) 2020 Stuart Swerdloff
```

Comment:
Protocol here is to always have the latest additions higher up the document, even if they are small. If people want to see specific contributions they can look at the git blame :)

https://github.com/pymedphys/pymedphys/blame/894260669bc847de39baec9a45646735bc5cddb9/pymedphys/_dicom/anonymise.py